.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
what is this file for ?

Reply:
this is husky which enables git hooks

Reply:
we need this, else husky files will get committed to git